projects/igniteui-angular/src/lib/simple-combo/simple-combo.component.ts

```diff
@@ -327,15 +327,23 @@ export class IgxSimpleComboComponent extends IgxComboBaseDirective implements Co
             // manually trigger text selection as it will not be triggered during editing
             this.textSelection.trigger();
             return;
-        }
-        if (event.key === this.platformUtil.KEYMAP.BACKSPACE
+        } else if (event.key === this.platformUtil.KEYMAP.BACKSPACE
             || event.key === this.platformUtil.KEYMAP.DELETE) {
             this._updateInput = false;
             this.clearSelection(true);
-        }
-        if (!this.collapsed && event.key === this.platformUtil.KEYMAP.TAB) {
+        } else if (event.key === this.platformUtil.KEYMAP.TAB) {
             this.clearOnBlur();
             this.close();
+        } else {
+            if (event.key !== this.platformUtil.KEYMAP.ARROW_DOWN && event.key !== this.platformUtil.KEYMAP.SPACE) {
+                const newFilterValue = this.filterValue + event.key.toLowerCase();
+                this.filterValue = newFilterValue;
```

Comment:
clicking on `control`, `alt` or any other special key and you will end up with `filtervalue` equal to `arrowupshiftaltcontrol` - this is not what we need here. Also, not sure how this will work with Japanese input !